geomesa-utils/pom.xml

```diff
@@ -157,6 +157,17 @@
                 </includes>
             </resource>
         </resources>
+
+        <plugins>
+            <plugin>
+                <groupId>pl.project13.maven</groupId>
```

Comment:
it was already in geomesa-utils, so anything using that should get it. Previously it just had the pom version - this adds in the git values.